packages/dapp-svelte-wallet/api/src/lib-wallet.js

```diff
@@ -158,21 +163,56 @@ export async function makeWallet({
       ...(depositBoardId && { depositBoardId }),
       brandPetname,
       pursePetname,
+      displayInfo: (issuerRecord && issuerRecord.displayInfo),
       value,
       currentAmountSlots: dehydratedCurrentAmount,
       currentAmount: fillInSlots(dehydratedCurrentAmount),
     };
     pursesState.set(purseKey, jstate);
+
     pursesFullState.set(
       purse,
       harden({
         ...jstate,
         purse,
         brand,
         actions: {
-          async send(receiverP, valueToSend) {
+          parseValue(str) {
```

Comment:
I think parsing formatted values on the wallet side is probably the wrong thing. We want the decimalPlaces to only be used for display, so this parsing should be done on the dapp side. The wallet code shouldn't care about it.